Foundation/NSSwiftRuntime.swift

```diff
@@ -87,7 +87,13 @@ extension ObjCBool : CustomStringConvertible {
 #endif
 
 internal class __NSCFType : NSObject {
+#if arch(x86_64) || arch(arm64) || arch(s390x) || arch(powerpc64) || arch(powerpc64le)
```

Comment:
Do we not have a word-size macro like C does here?

Reply:
This change also conflicts with a backout we've been trying to do for 32-bit that keeps getting shuffled, reopened, and closed, and expected to be re-opened again. And really, this particular use should be a _CFInfo, not a raw Int.

Reply:
Because this _cfinfo should match the size of _CFInfo

Reply:
See https://github.com/apple/swift-corelibs-foundation/pull/1693 (which is a followup of https://github.com/apple/swift-corelibs-foundation/pull/1689 which attempted a similar forward fix as you are applying to 32-bit with this change).